geograypher/meshes/meshes.py

```diff
@@ -162,19 +164,57 @@ def load_mesh(
             simplify_tol_meters=ROI_simplify_tol_meters,
         )
 
-        # Downsample mesh if needed
+        # Downsample mesh if needed transfer textures from original mesh to downsampled mesh
```

Comment:
You could say something more precise than "textures" if you want, like the active scalar, if it's point-based.